2/debian-10/rootfs/opt/bitnami/scripts/libkafka.sh

```diff
@@ -244,6 +248,16 @@ kafka_validate() {
         warn "The environment variables KAFKA_PORT_NUMBER and KAFKA_CFG_PORT are deprecated, you can specify the port number to use for each listener using the KAFKA_CFG_LISTENERS environment variable instead."
     fi
 
+    read -r -a users <<< "$(tr ',;' ' ' <<< "${KAFKA_CLIENT_USERS}")"
+    read -r -a passwords <<< "$(tr ',;' ' ' <<< "${KAFKA_CLIENT_PASSWORDS}")"
+    if [[ "${#users[@]}" -ne "${#passwords[@]}" ]]; then
+        print_validation_error "Specify the same number of passwords on KAFKA_CLIENT_PASSWORDS as the number of users on KAFKA_CLIENT_USERS!"
+    fi
+
+    if [[ "$protocol" = "SASL_PLAINTEXT" ]] || [[ "$protocol" = "SASL_SSL" ]]; then
```

Comment:
Minor improvement:

```suggestion
    if [[ "$protocol" = "SASL_PLAINTEXT" || "$protocol" = "SASL_SSL" ]]; then
```

Or:

```suggestion
    if [[ "$protocol" =~ ^SASL_(PLAINTEXT|SSL)$ ]]; then
```